### DISCUSSION TITLE
Bug on str_replace and paginate

### DISCUSSION THREAD

**Discussion by goashir** [laravel/framework#41640] [Q&A]
Hi,
in blade codes when have `paginate` and want use `str_replace ` laravel showing nothing!
**in Controller:**
```
 public function index()
    {
        $users = Product::paginate(15);
        return view('ghpage',['data'=>$users]);
    }
```

**in blade:**
```
<body dir="rtl" >
    <div>
        
    {{ $data->links() }}
</div>
```

**in tailwind.blade.php** (laravel/resources/views/vendor/pagination)
``` 
@if ($paginator->hasPages())
{!! str_replace('/?', '?',  $paginator->render() ) !!}
am i alive?
.
.
.
```
Laravel Framework 9.5.1
![Screenshot from 2022-03-23 22-19-25](https://user-images.githubusercontent.com/18716709/159763881-b3452075-a68e-438b-89e9-fc059bac5f2a.png)

**Comment by Kamandlou** [comment]
Add `Paginator::useTailwind();` to your `AppServiceProvider.php` boot method.
